smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude-traits.smithy

```diff
@@ -317,21 +320,25 @@ string title
 /// of constant values.
 @trait(selector: "string")
 @tags(["diff.error.add", "diff.error.remove"])
-map enum {
-    key: String,
-    value: EnumConstantBody
+list enum {
+    member: EnumDefinition
 }
 
 /// An enum definition for the enum trait.
 @private
-structure EnumConstantBody {
+structure EnumDefinition {
+    /// Defines the enum value that is sent over the wire.
+    @required
+    value: String,
```

Comment:
Since we're changing this anyway, we should add `@length(min: 1)`

Reply:
To be clear, I meant that the value shouldn't be an empty string, though the list for sure should not be empty either